db/migrate/20210428202726_add_people_in_household_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddPeopleInHouseholdToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :people_in_household, :integer, default: 1
+    User.all.each do |user|
```

Comment:
Normally this is not a good practise and i would think this will also fail, when you run rollback.